packages/backend/discovery/verifyConfigs.test.ts

```diff
@@ -0,0 +1,81 @@
+import { ChainId, ConfigReader } from '@l2beat/discovery'
+import { expect } from 'earl'
+import path from 'path'
+
+describe('Verify Discovery config files', () => {
+  // FIXME: This is a temporary hack to make the tests while configs are in backend
```

Comment:
I think we can remove this comment, since the context with which it was written is invalid now